exoservice-javascript/bin/exo-js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('../dist/cli');
+require('../dist/exo-js');
```

Comment:
This changes so that we are running the output of the webpack build. Here I named it `exo-js` and placed it in the `dist` folder, but it could be named anything we want it to be.